migrate.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash -e
 
-heroku run conjurctl db migrate
+heroku run possum db migrate
```

Comment:
does this undo a recent change? I feel like this might be connected to a recent issue